modules/ROOT/partials/configuration/allow_unsafe_link_target.adoc

```diff
@@ -1,7 +1,9 @@
+:blank: blank_
```

Comment:
Can we please call this `blanktarget` instead to match what was used in the 6.x docs? https://raw.githubusercontent.com/tinymce/tinymce-docs/staging/docs-6/modules/ROOT/partials/configuration/allow_unsafe_link_target.adoc

Reply:
done